api/pom.xml

```diff
@@ -203,7 +183,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.21.0</version>
                 <configuration>
-                    <skipTests>${skipTests}</skipTests>
+                    <skipTests>false</skipTests>
```

Comment:
Why did you make changes here?

Reply:
I think we shouldn't skip tests in any usual case. If a developer really wants it there is always `-DskipTests` or ` -Dmaven.test.skip=true`.

Reply:
Okay, I got it. Some of our customers build JAVA REST project in their repo to create jar.
Besides, mvn test is not working like this because we add system variables on MessageBirdClientTest class. Ou tests are mixed of unit and a bit integration tests. It would be nice if you revert this back.

Reply:
Done.